test_team_2.py

```diff
@@ -17,7 +33,19 @@ def test_nrp(self):
         """Test nrp functionality"""
 
     def test_url(self):
-        """Test url functionality"""
+        """Testing URL"""
+        #Positive test case, has PII in URL
+        test_url = "http://www.test.com/pageName?user=RealName&Password=TheRealPassword123"
+        expected = True
+        actual = check_url(test_url)
+        self.assertEqual(expected, actual)
+
+        #Negative test case, doesn't have PII in URL
```

Comment:
Another good test case would be to make sure only parameters are included.  The way your code is written that should be true, however, it is not tested.  For example, I would not expect www.secret.com to return a match, however, that is not tested and is an opportunity for improvement.

Reply:
Just added another test case for this example! Thanks!